tests/js/spec/stores/persistedStore.spec.jsx

```diff
@@ -0,0 +1,115 @@
+import {act} from 'react-test-renderer';
```

Comment:
@Neo-Zhixing was there any reason this test isn't in typescript? 👀

Reply:
i guess no... i've just changed the filename to `.spec.tsx`.

Reply:
Oh nice catch @dashed 🗡️